numpy/lib/function_base.py

```diff
@@ -2311,11 +2310,15 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
     array `m` and let ``f = fweights`` and ``a = aweights`` for brevity. The
     steps to compute the weighted covariance are as follows::
 
+        >>> m = np.arange(10)
+        >>> f = np.arange(10) * 2
+        >>> a = np.arange(10) ** 2.
         >>> w = f * a
         >>> v1 = np.sum(w)
         >>> v2 = np.sum(w * a)
-        >>> m -= np.sum(m * w, axis=1, keepdims=True) / v1
-        >>> cov = np.dot(m * w, m.T) * v1 / (v1**2 - ddof * v2)
+        >>> # NOTE: line below highly problematic in current NumPy
+        >>> m -= np.sum(m * w, axis=1, keepdims=True) / v1 # doctest: +SKIP
+        >>> cov = np.dot(m * w, m.T) * v1 / (v1**2 - ddof * v2) # doctest: +SKIP
```

Comment:
Maybe someone familiar with the covariance code has suggestions for the failing docstring cases above, currently marked with SKIP directives?

Reply:
Adjusted the example to produce the same result as `np.cov()` with the same input parameters. I had to define the degrees of freedom as that variable was missing in the code block.

Marking as resolved.